.github/workflows/build_and_deploy.yml

```diff
@@ -14,12 +14,13 @@ jobs:
   build_and_deploy:
     runs-on: ubuntu-latest
     env:
-      LATEX: lualatex
+      PDFTEX: xetex
     steps:
     - uses: actions/checkout@v3
     - uses: awalsh128/cache-apt-pkgs-action@v1
       with:
         packages: texinfo groff texlive-full
+        version: 1.0
```

Comment:
I saw this being used in the [`awalsh128/cache-apt-pkgs-action`](https://github.com/awalsh128/cache-apt-pkgs-action) and was hoping it would reduce the time spent on the caching actions, but it seems it doesn't.

I think it the line above could be removed without impact.